include/swift/AST/PropertyWrappers.h

```diff
@@ -170,16 +170,36 @@ void simple_display(llvm::raw_ostream &os, PropertyWrapperLValueness l);
 /// be initialized out-of-line using an expression of the wrapped property type.
 PropertyWrapperValuePlaceholderExpr *findWrappedValuePlaceholder(Expr *init);
 
-/// Describes the backing property of a property that has an attached wrapper.
-struct PropertyWrapperBackingPropertyInfo {
+/// The synthesized auxiliary declarations for a wrapped property.
+struct PropertyWrapperAuxiliaryVariables {
   /// The backing property.
   VarDecl *backingVar = nullptr;
 
   /// The synthesized projection property, if any. When present, this takes the name
   /// of the original wrapped property prefixed with \c $
   VarDecl *projectionVar = nullptr;
 
-private:
+  PropertyWrapperAuxiliaryVariables() {}
+
+  PropertyWrapperAuxiliaryVariables(VarDecl *backingVar, VarDecl *projectionVar)
+      : backingVar(backingVar), projectionVar(projectionVar) {}
+
+  /// Whether this is a valid property wrapper.
+  bool isValid() const {
+    return backingVar != nullptr;
+  }
+
+  explicit operator bool() const { return isValid(); }
+
+  friend bool operator==(const PropertyWrapperAuxiliaryVariables &lhs,
```

Comment:
We no longer use the AnyValue type eraser to store cached request values so unless you need it for some other purpose, you don’t have to overload operator== anymore.

Reply:
I removed it, thanks!